--- NVIDIA-Linux-x86-1.0-8774-pkg1/usr/share/applications/nvidia-settings.desktop~	2006-08-24 23:20:06.166925750 +0200
+++ NVIDIA-Linux-x86-1.0-8774-pkg1/usr/share/applications/nvidia-settings.desktop	2006-08-24 23:25:53.264618000 +0200
@@ -2,7 +2,9 @@
 Type=Application
 Encoding=UTF-8
 Name=NVIDIA X Server Settings
+Name[pl]=Ustawienia X serwera NVidii
 Comment=Configure NVIDIA X Server Settings
-Exec=__UTILS_PATH__/nvidia-settings
-Icon=__PIXMAP_PATH__/nvidia-settings.png
+Comment[pl]=Konfiguracja ustawień X serwera NVidii
+Exec=nvidia-settings
+Icon=nvidia-settings
 Categories=Application;System;
