--- NVIDIA-Linux-x86_64-180.35-pkg0/usr/share/applications/nvidia-settings.desktop.wiget	2009-02-23 06:01:15.000000000 +0100
+++ NVIDIA-Linux-x86_64-180.35-pkg0/usr/share/applications/nvidia-settings.desktop	2009-02-25 13:43:06.000000000 +0100
@@ -2,8 +2,9 @@
 Type=Application
 Encoding=UTF-8
 Name=NVIDIA X Server Settings
+Name[pl]=Ustawienia X serwera NVidii
 Comment=Configure NVIDIA X Server Settings
-Exec=__UTILS_PATH__/nvidia-settings
-Icon=__PIXMAP_PATH__/nvidia-settings.png
-Categories=Application;Settings;
-
+Comment[pl]=Konfiguracja ustawień X serwera NVidii
+Exec=nvidia-settings
+Icon=nvidia-settings
+Categories=Settings;HardwareSettings;
